python/tk_framework_desktopserver/certificates.py

```diff
@@ -274,7 +274,7 @@ def register(self):
 
         :returns: True on success, False on failure.
```

Comment:
This doesn't appear to be the case. Looks like no return and we get an exception on failure.